src/Jupyter/res/kernel.js

```diff
@@ -18,7 +18,7 @@ define(
                         {
                             token: "comment",
                             // include % to support kata special commands
-                            regex: /(\/\/|%).*/
+                            regex: /(\/\/|%kata).*/
```

Comment:
It might be good to include other magic commands (e.g. `%version`, `%simulate`) here as well.

Reply:
Sure! Updated the PR

Reply:
Also `%package`, `%workspace` and `%check_kata`